vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/patch_authenticator.go

```diff
@@ -178,34 +179,42 @@ func newAuthenticator(
 	// build cert authenticator
 	// TODO: add "system:" prefix in authenticator, limit cert to username
 	// TODO: add "system:" prefix to groups in authenticator, limit cert to group name
-	dynamicCA := certs.NewDynamicCA(apiClientCABundle)
-	if err := dynamicCA.CheckCerts(); err != nil {
+	clientCAContentProvider, err := dynamiccertificates.NewDynamicCAContentFromFile("client-ca-bundle", apiClientCABundle)
```

Comment:
You remove all of this file's content in favour of upstream, do we still need this commit?